### PR TITLE
VDB-1847 dog static storage transformer (Liq2.0)

### PR DESCRIPTION
**Pull request by elizabethengelman**
Based on #468

### CODE REVIEW

File: environments/mcdTransformers.toml [hunk 1]
```diff
@@ -3,7 +3,7 @@
...
...", "jug_file_ilk", "jug_file_vow", "jug_init", "log_median_price", "log_value", "median_diss_batch", "median_diss_single", "median_drop", "median_kiss_batch", "median_kiss_single", "median_lift", "new_cdp", "osm_change", "pot_cage", "pot_drip", "pot_exit", "pot_file_dsr", "pot_file_vow", "pot_join", "rely", "spot_file_mat", "spot_file_par", "spot_file_pip", "spot_poke", "tend", "tick", "vat_deny", "vat_file_debt_ceiling", "vat_file_ilk", "vat_flux", "vat_fold", "vat_fork", "vat_frob", "vat_grab", "vat_heal", "vat_hope", "vat_init", "vat_move", "vat_nope", "vat_rely", "vat_slip", "vat_suck", "vow_fess", "vow_file_auction_attributes", "vow_file_auction_address", "vow_flog", "vow_heal", "yank", "flip_bal_a_v1_1_14", "median_bal_v1_1_14", "flip_yfi_a_v1_1_14", "median_yfi_v1_1_14", "flip_gusd_a_v1_1_5", "flip_renbtc_a_v1_2_1", "flip_uni_a_v1_2_1", "median_uni_v1_2_1", "flip_aave_a_v1_2_2", "median_aave_v1_2_2", "flip_univ2daieth_a_v1_2_2", "flip_univ2wbtceth_a_v1_2_4", "flip_univ2usdceth_a_v1_2_4", "flip_univ2daiusdc_a_v1_2_5", "flip_univ2ethusdt_a_v1_2_5", "flip_univ2unieth_a_v1_2_6", "flip_univ2linketh_a_v1_2_6", "flip_univ2aaveeth_a_v1_2_7", "flip_univ2wbtcdai_a_v1_2_7", "dog_bark"]
```
Comment (elizabethengelman) [inline environments/mcdTransformers.toml:6]: 🚧 need to change "dog_v1_x_x"

File: environments/mcdTransformers.toml [hunk 2]
```diff
@@ -92,6 +92,12 @@
     rank = "0"
     repository = "github.com/makerdao/vdb-mcd-transformers"
     type = "eth_event"
+  [exporter.dog_v1_x_x]
```
Comment (elizabethengelman) [inline environments/mcdTransformers.toml:102]: 🚧 need to change "dog_v1_x_x" and the path

File: plugins/execute/transformerExporter.go [hunk 3]
```diff
@@ -223,6 +224,7 @@ func (e exporter) Export() ([]event.TransformerInitializer, []storage.Transforme
 			cat_v1_0_0.StorageTransformerInitializer,
 			cat_v1_1_0.StorageTransformerInitializer,
 			cdp_manager.StorageTransformerInitializer,
+			dog_v1_x_x.StorageTransformerInitializer,
```
Comment (elizabethengelman) [inline plugins/execute/transformerExporter.go:227]: 🚧 need to change "dog_v1_x_x"

File: transformers/component_tests/initializers/storage/storage_initializers_test.go [hunk 4]
```diff
@@ -68,6 +70,13 @@ var _ = Describe("Storage transformer initializers", func() {
 		Expect(transformer.GetContractAddress().String()).To(Equal(address))
 	})
 
+	It("configures the dog v1_x_x", func() {
```
Comment (elizabethengelman) [inline transformers/component_tests/initializers/storage/storage_initializers_test.go:73]: 🚧 need to change "dog_v1_x_x"

File: transformers/storage/dog/initializers/v1_x_x/initializer.go [hunk 5]
```diff
@@ -0,0 +1,18 @@
+package v1_x_x
```
Comment (elizabethengelman) [inline transformers/storage/dog/initializers/v1_x_x/initializer.go:1]: 🚧 need to change "v1_x_x"

File: transformers/storage/dog/initializers/v1_x_x/initializer.go [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+import (
+	"github.com/ethereum/go-ethereum/common"
+	mcdStorage "github.com/makerdao/vdb-mcd-transformers/transformers/storage"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/storage/cat"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/storage/dog"
+	"github.com/makerdao/vdb-transformer-utilities/pkg/shared/constants"
+	"github.com/makerdao/vulcanizedb/libraries/shared/factories/storage"
+)
+
+var dogAddress = constants.GetContractAddress("MCD_Dog_1_x_x")
```
Comment (elizabethengelman) [inline transformers/storage/dog/initializers/v1_x_x/initializer.go:12]: 🚧

**Review by yaoandrew — APPROVED** [review]

**Review by ecmendenhall — APPROVED** [review]

**Pull request merged**